samples/vmc/networks_nsxv/nat_rule_crud.py

```diff
@@ -29,35 +29,26 @@ class NatRuleCrud(object):
     """
 
     def __init__(self):
-        parser = argparse.ArgumentParser(
-            formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-        parser.add_argument(
-            '-r',
-            '--refresh-token',
-            required=True,
-            help='VMware Cloud API refresh token')
+        required_args.add_argument(
+            '--org-id', required=True, help='Organization identifier.')
```

Comment:
Most of the samples require org_id and sddc_id. Please check if we can move both to sample_cli.py and build seperate arg parser where it is not required